File: code/cb178.pas
program Bai178;
var n, k: integer;
    dem, i: byte; 
  begin
  readln(n);
  k := 0;
  while n > 0 do 
    begin
    k := 10*k + n mod 10;
    n := n div 10;
    end;
    if  k mod 13 = 0 then write('YES') else write('NO') ;
 end.

File: code/cb178.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,k;
    cin>>n;
    k=0;
    while (n>0){
        k=10*k+n%10;
        n=n/10;
    }
    if (k%13==0)
        cout<<"YES";
    else cout<<"NO";
    return 0;
}